dm/master/scheduler/scheduler.go

```diff
@@ -1282,7 +1301,7 @@ func (s *Scheduler) tryBoundForWorker(w *Worker) (bounded bool, err error) {
 	}()
 
 	// 3. try to bound them.
-	err = s.boundSourceToWorker(source, w)
+	err = s.boundSourceToWorker(source, w, s.sourceCfgs[source].EnableRelay)
```

Comment:
Where is the logic that a start-relay worker will not bound to another source?

Reply:
this PR didn't introduce `start-relay`, so no such problems.